torchvision/datasets/video_utils.py

```diff
@@ -131,16 +137,16 @@ def __init__(
             self._init_from_metadata(_precomputed_metadata)
         self.compute_clips(clip_length_in_frames, frames_between_clips, frame_rate)
 
-    def _compute_frame_pts(self):
+    def _compute_frame_pts(self) -> None:
         self.video_pts = []
         self.video_fps = []
 
         # strategy: use a DataLoader to parallelize read_video_timestamps
         # so need to create a dummy dataset first
         import torch.utils.data
 
-        dl = torch.utils.data.DataLoader(
-            _VideoTimestampsDataset(self.video_paths),
+        dl: torch.utils.data.DataLoader = torch.utils.data.DataLoader(
```

Comment:
Why do we need to annotate `dl` here?

Reply:
mypy is throwing me errors otherwise:
```
torchvision/datasets/video_utils.py:150: error: Need type annotation for 'dl'  [var-annotated]
            dl = torch.utils.data.DataLoader(
                 ^
```

Reply:
Weird, but ok.